torchbox/torchbox.com
# Work - results/stats blocks - BE

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1367490573)

### Description of Changes Made

This builds up on #57 by defining the `NumericStatisticsBlock` and `TextualStatisticsBlock` and ensuring their availability in the `WorkStoryBlock`.

### How to Test

When you create a Work Page, you should be able to add these stats blocks and see them appear on the template.

### Screenshots

<details>
  <summary>Expand to see more</summary>

![image](https://github.com/torchbox/torchbox.com/assets/7713776/fdd6e017-8e57-4936-81dc-1f6c24e3d107)

![image](https://github.com/torchbox/torchbox.com/assets/7713776/31ed6311-e04a-4944-bc51-7308f3b64529)


</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on Ubuntu
  - Latest version of Firefox on Ubuntu

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
